connector/oidc/oidc.go

```diff
@@ -341,6 +345,7 @@ func (c *oidcConnector) createIdentity(ctx context.Context, identity connector.I
 		return identity, errors.New("oidc: no id_token in token response")
 	}
 
+	c.logger.Debugf("Claims::: %q \n", claims)
```

Comment:
I assume this stayed here by accident?

Reply:
Actually this came as a need when we were recently debugging stuff on a production environment where we did not have access to the IdP. It became painful to prove that some of the claims that we were expecting were not being sent by the IdP. But if you think it is unnecessary I will remove it.

Reply:
Removed this in 345eba905ddb2664855e79da2acf816ad6b97eb5